src/main.js

```diff
@@ -435,7 +435,7 @@ Formsy.Form = React.createClass({
   render: function () {
 
     return (
-      <form {...this.props} onSubmit={this.submit}>
+      <form id={this.props.id} {...this.props} onSubmit={this.submit}>
```

Comment:
If we just hold our breaths a little bit more I think React 0.14 is right around the corner :-) What we could do though is update the current readme encouraging people to use the new branch with React 0.14 Beta. That helps Facebook testing new React and us testing new Formsy.

I think most issues now can be worked around until next release.